pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -4,7 +4,7 @@
 , freetype, fontconfig, libXft, libXrender, libxcb, expat
 , libuuid
 , libxml2
-, glib, gtk3, pango, gdk-pixbuf, cairo, atk, at-spi2-atk, at-spi2-core
+, glib, gtk3, gtk4, pango, gdk-pixbuf, cairo, atk, at-spi2-atk, at-spi2-core
```

Comment:
A similar issue exists with qt: currently vivadi is built against qt5 and doesn't work with qt6: https://github.com/NixOS/nixpkgs/pull/286522#issuecomment-1969058722

I am somewhat hesitant making vivaldy unconditionally depend on all of gtk3, gtk4, qt5, and qt6.

Maybe a better approach is to make both of these parameters?

```nix
, gtk ? gtk3
, qt ? qt5
```

Reply:
We don't depend on gtk2 directly and we probably can't depend on qt5 and qt6 at the same time anyway.

Reply:
Yeah, this does seem to work for me locally:

https://github.com/NixOS/nixpkgs/pull/292148

Don't understand enough about nixpkgs to judge whether this is a good idea or not :)